auth0_flutter_web/lib/src/options.dart

```diff
@@ -118,6 +118,40 @@ class GetTokenWithPopupOptions extends PopupLoginOptions {
   });
 }
 
+@JS()
+@anonymous
+class GetTokenSilentlyOptions {
+  external String get audience;
+  external bool get ignoreCache;
+  external String get redirect_uri;
+  external String get scope;
+  external int get timeoutInSeconds;
```

Comment:
`double` is used for other timeout field [like this](https://github.com/anthonychwong/auth0-flutter-web/blob/e201e95ad5bbbff097e819f4b2db94440a6458a7/auth0_flutter_web/lib/src/options.dart#L55), since it aligns to the type `number` in official sdk [link](https://github.com/auth0/auth0-spa-js/blob/4a9c17359ec9585d0a379d5f601ab65e27c7efdb/src/global.ts#L219).

I would like to know if it is better to use `int` for timeout fields.

Reply:
I'm wondering if we could use `num` instead of choosing one way or another - what do you think?
In any case, this is also applicable to `timeoutInSeconds` in `PopupConfigOptions` below - will fix both per your decision.

Reply:
Thanks, `num` looks good here.